docs/config/development/feature-from-scratch/index.rst

```diff
@@ -0,0 +1,385 @@
+          type: PADDED_GRID
+          width: 12
+          padding: 4
+          salt: 5864
+
+    The ``PADDED_GRID`` distributor type utilizes cells in a grid with
+    the feature placed within each cell with padding between each cell
+    to ensure that features don't generate too close to one another.
+
+    ``PADDED_GRID`` utilizes the parameters ``width``, ``padding``, and ``salt``.
+
+    * ``Width`` - Determines the size of each cell that will contain your feature
+    * ``Padding`` - Determines the gap between each cell
+    * ``Salt`` - Offsets the results of the distributor to prevent overlap
+
+5. Create the feature locator
+-----------------------------
+
+.. card::
+
+    Locators determine the y-axis placement of a feature in the world.
+
+    Add the ``config-locators`` addon to the pack manifest, using versions ``1.+``.
+
+    This addon will allow us to create locators within feature config files.
+
+    .. code-block:: yaml
+        :caption: pack.yml
+        :linenos:
+        :emphasize-lines: 7
+
+        id: YOUR_PACK_ID
+
+        version: 0.1.0
+
+        addons:
+          ...
+          config-locators: "1.+"
+
+    Configure the ``oak_trees.yml`` config to utilize the ``TOP`` locator type as shown below.
+
+    .. code-block:: yaml
+        :caption: oak_trees.yml
+        :linenos:
+        :emphasize-lines: 7-11
+
+        id: OAK_TREES
+        type: FEATURE
+
+        distributor:
+          ...
+
+        locator:
+          type: TOP
+          range:
+            min: 0
+            max: 319
+
+    The ``TOP`` locator type will place the feature on the block located at the highest possible y-level.
+
+.. tip::
+
+    You can utilize multiple locators for stricter criteria as shown below with the ``AND`` locator.
+
+    .. code-block:: yaml
+        :caption: feature.yml
+        :linenos:
+        :emphasize-lines: 2
+
+        locator:
+          type: AND
+          locators:
+            - type: TOP
+              range: &range  #range values anchored for other locators to use
+                min: 0
+                max: 319
+            - type: PATTERN
+              range: *range  #references previously anchored range values
+              pattern:
+                type: MATCH_SET
+                blocks:
+                  - minecraft:grass_block
+                  - minecraft:dirt
+                offset: -1
+
+6. Apply the structure
+----------------------
+
+.. card::
+
+    You can now add your structure to the ``oak_trees.yml`` config with the highlighted lines below.
+
+    .. code-block:: yaml
+        :caption: oak_trees.yml
+        :linenos:
+        :emphasize-lines: 10-13
+
+        id: OAK_TREES
+        type: FEATURE
+
+        distributor:
+          ...
+
+        locator:
+          ...
+
+        structures:
+          distribution:
+            type: CONSTANT
+          structures: oak_tree
+
+    The ``structures`` sub-configuration consists of a structure or list of structures for
```

Comment:
Keep consistent terminology in reference to *parameters* rather than terms like sub-configuration, in this case you could make specific reference to `structures.distribution` and `structures.structures`, I might comb through the docs eventually:tm: and make links to the parameters section in 'The Config System' page whenever that term is used, linking to that here would probably be good too